app/views/devices_guidance/preparing_4g_wireless_routers.md

```diff
@@ -47,11 +50,25 @@ When distributing devices, you should inform families, young people and care lea
+
+This filtering system:
+- blocks a range of content categories
+- limits searching to the ‘safe search’ supported by popular search engines
+- should not prevent legitimate use of the devices
+
+If a child is able to access inappropriate content, you should report this through the [Support Portal](https://computacenterprod.service-now.com/dfe).
+
+## Safeguarding resources for schools, colleges, families, carers, and young people
+
+These resources include detailed guidance for schools and colleges on safe remote education, virtual lessons, and live streaming. There's also information on how to make sure children and young people are safe online.
 
-The device will connect to the mobile network with the strongest signal available. Regardless of which network it uses, all traffic will pass through a private, secure connection to a filtered service. This filtering system blocks a range of content categories and limits searching to the ‘Safe Search’ supported by popular search engines. This filtering should not prevent legitimate use of the devices. If a child is able to access inappropriate content, you should report this through the [Support Portal](https://computacenterprod.service-now.com/dfe).
+- [Keeping Children Safe in Education](https://www.gov.uk/government/publications/keeping-children-safe-in-education--2) - guidance that schools and colleges must follow when carrying out their duties to safeguard and promote the welfare of children – both online and offline.
+- [Safeguarding and remote education during coronavirus](https://www.gov.uk/guidance/safeguarding-and-remote-education-during-coronavirus-covid-19) - guidance to help you understand safeguarding procedures when planning remote education strategies and teaching remotely.
+- [Support for parents and carers to keep children safe from online harm](https://www.gov.uk/government/publications/coronavirus-covid-19-keeping-children-safe-online/coronavirus-covid-19-support-for-parents-and-carers-to-keep-children-safe-online) - outlines resources to help keep children safe from different risks online, and where to go to receive support and advice
+- [Support to stay safe online](https://www.gov.uk/guidance/covid-19-staying-safe-online - includes information on security and privacy settings
 
-Parents and guardians of the young people making use of these devices should be mindful of advice on supervising internet use by children. You may wish to send the following information to families when distributing 4G wireless routers:
+In conversations with parents, carers, and children you should emphasise the importance of a safe online environment and offer support and advice on how to do this. It is also important that children, parents, carers, and school staff are clear on what the reporting mechanisms are if they have any concerns regarding online safety.
```

Comment:
... how to create this.

It's also important ...

... clear on how to report any concerns they may have about online safety.